src/index.d.ts

```diff
@@ -185,24 +171,10 @@ export interface AvatarProps {
   iconStyle?: StyleProp<TextStyle>;
 
   /**
-   * Small sized icon
+   * Size of Avatar
```

Comment:
Just one small change!

Add a new line here and put `@default "small"`

Reply:
sure